examples/musicdiff_demo.ipynb

```diff
@@ -0,0 +1,533 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "musicdiff Demo",
+      "provenance": [],
+      "authorship_tag": "ABX9TyMLSYdH36tL7M7GJLDqErxX",
+      "include_colab_link": true
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "view-in-github",
+        "colab_type": "text"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/gregchapman-dev/musicdiff/blob/return_output_path/examples/musicdiff_demo.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 1,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "Xs5PXxMtwcoH",
+        "outputId": "95aabe72-90dd-411f-9d6d-8020dd758315"
+      },
+      "outputs": [
```

Comment:
All these outputs are not here for the benefit of anyone using this colab.  Can they be removed somehow?